servicemodels/test_sm_aper_go_lib/v1/test-sm-ies/test_sm.pb.go

```diff
@@ -1,3 +1,12 @@
+//
+//SPDX-FileCopyrightText: 2020-present Open Networking Foundation <info@opennetworking.org>
+//
+//SPDX-License-Identifier: Apache-2.0
+
+////////////////////// test-sm-ies.proto //////////////////////
+// Protobuf generated from /tes_sm.asn1 by asn1c-0.9.29
+// TEST-SM-IEs { iso(1) identified-organization(3) dod(6) internet(1) private(4) enterprise(0) e2(1) version1(1) e2sm(1) test-sm-IEs(0) }
+
```

Comment:
Same as in previous PR - is this file included in this PR by intent or it's just a leftover? In case of the latter, please exclude it from the PR.